CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Beginning in Cromwell 81 we will populate the `tags` field of tasks created by t
 with the labels applied to the workflow at creation time.  No guarantee is made about labels
 added while the workflow is running.
 
+### Batch Compute Service backend removed
```

Comment:
This should maybe say 'Alibaba' somewhere? IMHO it sounds like a broader change than it actually is.

Reply:
+100